### DISCUSSION TITLE
安装中pypi换源

### DISCUSSION THREAD

**Discussion by enliliu** [datawhalechina/learn-python-the-smart-way-v2#28] [提问答疑]
您好助教，我在安装过程中pipy换源的时候遇到了如下问题：
![pipy换源](https://github.com/datawhalechina/learn-python-the-smart-way-v2/assets/121369353/ba935316-7d3b-4d09-b835-2cd8ec16c66e)

我在百度上查到在环境变量path中添加pip的路径，但是没有效果
![2](https://github.com/datawhalechina/learn-python-the-smart-way-v2/assets/121369353/816b30fe-a6b1-4e86-90d1-aca79cfc9a51)

![1](https://github.com/datawhalechina/learn-python-the-smart-way-v2/assets/121369353/718bfd08-bdc5-47d3-9a3c-6171b3592775)
然后我在miniconda3的文件夹中添加了一个pip文件夹，里面添加了了pip.int文件：
![3](https://github.com/datawhalechina/learn-python-the-smart-way-v2/assets/121369353/3e4d398e-bbff-4e37-833b-6cca80ec3df3)
我想问一下这样可以吗，万分感谢

**Comment by anine09** [comment]
Hi，@enliliu 你能在终端中查看 Python 版本吗？我们发现你的环境变量特别混乱，似乎是由于你手动添加导致的

**Comment by enliliu** [comment]
Hi,助教
我之前下载过python软件，但刚才我把之前下载过的彻底清理了一遍，现在pypi换源没有问题了，但是又出现了新问题：我刚打开anaconda powoshell prompt就会提示字符串缺少终止符，我卸载又重新安装了anaconda，还是没有解决，我在百度上查,把ps1文件内容中字符串后面的引号修改为\""，还是没解决，我现在好懵，感谢助教






                       
原始邮件
                       
                     

发件人："Epsilon Luoo"< ***@***.*** &gt;;

发件时间：2023/12/9 15:35

收件人："datawhalechina/learn-python-the-smart-way-v2"< ***@***.*** &gt;;

抄送人："enliliu"< ***@***.*** &gt;;"Mention"< ***@***.*** &gt;;

主题：Re: [datawhalechina/learn-python-the-smart-way-v2] 安装中pypi换源 (Discussion #28)




 
***@***.*** 你能在终端中查看 Python 版本吗？我们发现你的环境变量特别混乱，似乎是由于你手动添加导致的

**Comment by enliliu** [comment]
Hi，助教，新问题我自己解决了，万分感谢






                       
原始邮件
                       
                     

发件人："Epsilon Luoo"< ***@***.*** &gt;;

发件时间：2023/12/9 15:35

收件人："datawhalechina/learn-python-the-smart-way-v2"< ***@***.*** &gt;;

抄送人："enliliu"< ***@***.*** &gt;;"Mention"< ***@***.*** &gt;;

主题：Re: [datawhalechina/learn-python-the-smart-way-v2] 安装中pypi换源 (Discussion #28)




 
***@***.*** 你能在终端中查看 Python 版本吗？我们发现你的环境变量特别混乱，似乎是由于你手动添加导致的
